From 464e30e2c61654a9423a6bebc061a13bfa6a4d91 Mon Sep 17 00:00:00 2001
From: Viktoria Maximova <viktoria.maksimova@intel.com>
Date: Fri, 28 Apr 2023 07:29:16 -0700
Subject: [PATCH 44/79] [NFC][DebugInfo] Rewrite NonSemantic/DebugInfoProducer
 test

There is a bug in -r -spirv-text mode in translation of Strings with spaces.
Temporary avoid running such pipeline until proper fix is delivered.
---
 test/DebugInfo/NonSemantic/DebugInfoProducer.ll | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/DebugInfo/NonSemantic/DebugInfoProducer.ll b/test/DebugInfo/NonSemantic/DebugInfoProducer.ll
index 9de3cc59..49a894f6 100644
--- a/test/DebugInfo/NonSemantic/DebugInfoProducer.ll
+++ b/test/DebugInfo/NonSemantic/DebugInfoProducer.ll
@@ -20,7 +20,8 @@
 ; Check that we don't produce "producer" info for NonSemantic.Shader.DebugInfo.100 as it's not specification conformant
 ; RUN: llvm-spirv %t.bc --spirv-debug-info-version=nonsemantic-shader-100 -spirv-text -o %t.spt
 ; RUN: FileCheck %s --input-file %t.spt --check-prefix CHECK-SPIRV-100
-; RUN: llvm-spirv -spirv-text -r -emit-opaque-pointers %t.spt -o %t.rev.bc
+; RUN: llvm-spirv %t.bc --spirv-debug-info-version=nonsemantic-shader-100 -o %t.spv
+; RUN: llvm-spirv -r -emit-opaque-pointers %t.spv -o %t.rev.bc
 ; RUN: llvm-dis %t.rev.bc -o - | FileCheck %s --check-prefix CHECK-LLVM-100
 
 ; ModuleID = 's.bc'
-- 
2.20.1

